Source/Shaders/PostProcessFilters/FXAA3_11.glsl

```diff
@@ -0,0 +1,2074 @@
+/**
```

Comment:
If I move it out of the `Shaders` directory, it won't be part of the build. So it needs to stay there or I can modify the build to check the `ThirdParty` directory.